src/main/java/no/finn/unleash/variant/VariantUtil.java

```diff
@@ -61,7 +62,10 @@ private static String getIdentifier(UnleashContext context) {
     }
 
     public static Variant selectVariant(
-            FeatureToggle featureToggle, UnleashContext context, Variant defaultVariant) {
+        @Nullable FeatureToggle featureToggle, UnleashContext context, Variant defaultVariant) {
+        if(featureToggle == null) {
```

Comment:
Annotations revealed that the `featureToggle` could be passed in as a null, so adding this check to account for that. Please check that returning the `defaultVariant` is the correct response.

Reply:
Looking at the places we call this, I'd say this should be the correct response